textsubs.lic

```diff
@@ -47,6 +47,12 @@ if get_settings.integrate_shit_list_with_textsubs && File.file?("#{checkname}_sh
   end
 end
 
+unless get_settings.private_textsubs.empty?
```

Comment:
Not necessary for this PR, but it would be nice to cache `get_settings` instead of calling it 4+ times in this script.

Reply:
I have thought about that, and I did at first until I realize those were all outside of the textsubs class. How do you want me to handle it?

Reply:
It can still be stored off into a var. But that's a problem for another day.

Reply:
OK, I pushed it!

Reply:
No clue why I was brain farting on the variable simply outside of the class, I'm doing too many things at once.